src/nvim/screen.c

```diff
@@ -2661,7 +2660,7 @@ win_line (
     extra_check = true;
   }
 
-  int sign_idx = 0;
+  int sign_idx = 0, sign_cells = win_signcol_width(wp);
```

Comment:
we generally do not declare multiple symbols on one line.  Specially for non-constant initializer.